dbt_project.yml

```diff
@@ -30,7 +30,7 @@ vars:
   path_lookback_days: 30 # Restrict to marketing channels within this many days of the conversion (recommended: 30 / 14 / 7)
   path_transforms: [['Exposure', null]] # array of path transforms (and their arguments) to perform on the full conversion path (see udfs.sql)
   consider_intrasession_channels: false # false = only considers the channel at the start (first page view) of the session, true = consider multiple channels in the conversion session as well as historical channels
-
+  channels_to_exclude: ['Direct', 'Unmatched_Channel'] # Channels to exclude before creating path summaries (and therefore excluded from fractribution analysis)
```

Comment:
Let's keep the default as an empty list and then have a comment that shows how you would exclude just direct i.e.,

`# channels_to_exclude: ['Direct']`

Reply:
Yep sure